conversion/conversion.go

```diff
@@ -0,0 +1,789 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Package conversion handles initial setup for the command line tool
+// and web APIs.
+
```

Comment:
Package comments should appear immediately before the package declaration i.e. 
- line 14: should just be a blank line (get rid of //)
- line 17: delete this line

Reply:
updated as mentioned